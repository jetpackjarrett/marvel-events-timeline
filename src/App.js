import React from 'react';
import Timeline from './timeline/Timeline';
import './app.css';

const App = () => {
  return (
    <div className="app">
      <header className="app-header">
        <h1>Timeline of Marvel Events</h1>
      </header>
      <main>
        <Timeline />
      </main>
      <footer className="app-footer">
        <p>
          <a
            rel="noopener noreferrer"
            target="_blank"
            href="http://marvel.com">
            Data provided by Marvel. &copy; 2017 MARVEL
          </a>
        </p>
      </footer>
    </div>
  );
};

export default App;
