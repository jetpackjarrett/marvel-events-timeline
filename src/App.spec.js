/*eslint import/no-extraneous-dependencies: ["error", { "devDependencies": true }] */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './App';

describe('App', () => {
  it('renders the App', () => {
    const component = shallow(
      <App />
    );
    expect(toJson(component)).toMatchSnapshot();
  });
});
