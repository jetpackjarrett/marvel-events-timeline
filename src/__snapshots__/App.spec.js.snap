exports[`App renders the App 1`] = `
<div
  className="app">
  <header
    className="app-header">
    <h1>
      Timeline of Marvel Events
    </h1>
  </header>
  <main>
    <Timeline />
  </main>
  <footer
    className="app-footer">
    <p>
      Data provided by Marvel. Â© 2014 Marvel
    </p>
  </footer>
</div>
`;
